packages/material-ui/src/Button/Button.test.js

```diff
@@ -270,7 +268,13 @@ describe('<Button />', () => {
   });
 
   it('should render a button with startIcon', () => {
-    const { getByRole } = render(<Button startIcon={<span>icon</span>}>Hello World</Button>);
+    let getByRole;
+
+    expect(() => {
+      const { getByRole: getByRoleLocal } = render(<Button startIcon={<span>icon</span>}>Hello World</Button>);
```

Comment:
Emotion throws this error when `":first-child"` is used.

Reply:
Is it because emotion injects <style> elements in the page when SSR isn't configured?

Reply:
https://github.com/emotion-js/emotion/issues/1178

Reply:
Exactly, but as we are following the advanced SSR configuration - https://emotion.sh/docs/ssr#advanced-approach we should be fine.

Reply:
Managed to avoid the warning with https://github.com/mui-org/material-ui/pull/24107/commits/745e163a761e4a34e1f5ac5aa22956e0d22b748b. See comment - https://github.com/mui-org/material-ui/pull/24107#discussion_r548063529